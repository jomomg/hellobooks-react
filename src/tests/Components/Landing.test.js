import React from 'react';
import LandingPage from '../../components/landing';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('<LandingPage/>', ()=>{
    test('it renders successfully', ()=>{
        const landingPage = shallow(<LandingPage/>);
        expect(shallowToJson(landingPage)).toMatchSnapshot();
    });
});