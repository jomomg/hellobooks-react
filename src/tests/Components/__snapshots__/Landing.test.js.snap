// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage/> it renders successfully 1`] = `
<Fragment>
  <div
    style={
      Object {
        "backgroundColor": "teal",
        "margin": "0",
      }
    }
  >
    <div
      className="container"
    >
      <br />
    </div>
  </div>
  <div
    style={
      Object {
        "backgroundColor": "teal",
        "margin": "0",
      }
    }
  >
    <div
      className="container"
    >
      <p
        className="logo"
      >
        Hello Books
      </p>
      <Route
        render={[Function]}
      />
      <Route
        render={[Function]}
      />
      <span
        style={
          Object {
            "color": "white",
            "fontFamily": "Courier New",
            "fontStyle": "bold",
          }
        }
      >
        or
      </span>
      <Route
        render={[Function]}
      />
    </div>
  </div>
  <footer
    className="page-footer teal"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col l6 s12"
        >
          <h4
            className="white-text"
          >
            Welcome to Hello Books
          </h4>
          <h5
            className="grey-text text-lighten-4"
          >
            Here you can borrow a book or two from our collection, enjoy it, return it, and repeat the process
          </h5>
        </div>
      </div>
    </div>
    <div
      className="footer-copyright"
    >
      <div
        className="container"
      >
        Â© 2018 Jomo Gitau
      </div>
    </div>
  </footer>
</Fragment>
`;
